src/utils.js

```diff
@@ -102,6 +102,29 @@ const removeAlreadyChallengedPlayers = (players, playerId, in_progress) => {
 
 const removeEmptyArray = array => array.filter(i => (i.length === 0) ? false : true)
 
+const factorKCaculator = elo => {
+    switch (elo) {
+        case (elo < 2100):
+            return 32
+        case (elo >= 2100 && elo <= 2400):
+            return 24
+        case (elo > 2400):
+            return 16
+        default:
+            return 32
+    }
+}
```

Comment:
The `factorKCaculator` isn't working properly, check this CodePen:

https://codepen.io/Xotl/pen/MWgNKEO?editors=1012

I added a suggestion named `newFactorKCaculator`.